src/js/common/schemaform/ObjectField.jsx

```diff
@@ -210,7 +211,7 @@ class ObjectField extends React.Component {
             {DescriptionField && <DescriptionField formData={formData} formContext={formContext} options={uiSchema['ui:options']}/>}
             {!textDescription && !DescriptionField && description}
           </div>}
-          {uiOptions.showPrefillMessage && formContext.prefilled && <PrefillMessage/>}
+          {process.env.BRANCH_NAME !== 'production' && uiOptions.showPrefillMessage && <PrefillMessage message={prefillMessage}/>}
```

Comment:
I don't think `process.env.BRANCH_NAME` name is right, I think it's `__BUILDTYPE__` you want.

Reply:
Thanks @jbalboni, I've also reorganized this a bit more now so that the conditions reflect the different behaviors/scenarios.
- If in production or not on the military service page, check that information has been prefilled (in addition to uiOptions.showPrefillMessage being true).
- If not in production, and on the military service page, show the prefill notification as long as uiOptions.showPrefillMessage is true (information needs not be prefilled).

Reply:
Does that work for you @stacyhlinn?